benchmarks/DeepLearning/Layers/RMSNorm/GoogleBenchmarkMain.cpp

```diff
@@ -0,0 +1,147 @@
+//===- GoogleBenchmarkMain.cpp---------------------------------------------===//
```

Comment:
Recommand to centralize this title and keep the number of characters per line to 80.

Reply:
fixed

Reply:
In this version of the code standard, the title remains not centered, otherwise it will be too troublesome to check each file and centralize the tiles. Please help me change it back. Thanks~